themes/contribute.jquery.org/cla-check.php

```diff
@@ -110,7 +110,14 @@ function getProcessedPost( $data ) {
 function neglectedAuthors( $data ) {
 	$html = "<ul>\n";
 	foreach ( $data->data->neglectedAuthors as $author ) {
-		$html .= "<li>" . htmlspecialchars( "$author->name <$author->email>" ) . "</li>\n";
+		$html .= "<li>" . htmlspecialchars( "$author->name <$author->email>" );
+		if ( count( $author->errors ) ) {
+			$html .= ": ";
+			foreach( $author->errors as $error ) {
```

Comment:
We do care, we just don't have automated tests. There should be a space on this line. Also, for strings that don't require interpolation, single quotes should be used.